mzegar/PoseBook
# Add "Submit Review" Popup

**Description**

- Add and integrate submit review popup into main screen.

- **Note:** 
- Currently location & sub-location are hardcoded. We will need to integrate location from the photo in the future. 
- Web call to firebase to submit review & record creation are yet to be implemented.
- Currently, the popup is shown directly after taking the 2nd photo. We will need to move it to the share button when those are implemented.

**To Test:**

1. Run the app.
2. Take a photo and it will should the looks good button.
3. Click on looks good button and take a photo again.
4. Verify the review popup is shown.
5. Verify the design matches the one we had on Figma.
6. Type something into the text box and verify its functionality.
7. Tap on rating stars and verify its functionality.
8. Click on submit review and verify the popup dismisses and the toast "Review Submitted" appears.
9. Repeat 1 - 4.
10. Tap on the cancel button and verify the popup dismisses.
